### PR TITLE
Wqp 1819 - Fix menus/tooltips that were blocked by accordion header

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Fix menus/tooltips that were blocked by accordion header

Description
-----------
I also revamped the header to use USWDS header components and removed a lot of the custom style for that. I think to solve the caret color problem we will need to investigate using themed colors and/or USWDS mixins.

I separated out the advanced form and the basic form into their own partials and simplified the advanced form so that there is less nesting and the nesting that does exists makes sense.

The blocking problem was solved by setting overflow: visible for the accordion contents.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/scss/partials/_portal-form.scss [hunk 1]
```diff
@@ -74,12 +75,17 @@
 
 #params {
   .wqp-textarea {
-    @include u-height(8);
-    width: 100%;
+    @include u-height(10);
+    max-width: 100%;
```
Comment (abriggs-usgs) [inline assets/scss/partials/_portal-form.scss:79]: could use @include u-maxw('full'), I think

File: assets/scss/partials/_portal-form.scss [hunk 2]
```diff
@@ -116,34 +128,38 @@
...
+  #current-url-textarea {
+    @include u-margin-bottom('105');
+  }
 
-    .share-container, #query-box-text {
-      width: 100%;
-      @include u-padding-top('105');
-    }
+  .share-container, #query-box-text {
+    width: 100%;
```
Comment (abriggs-usgs) [inline assets/scss/partials/_portal-form.scss:159]: @include u-width('full)?

File: server/wqp/portal_ui_blueprint/templates/partials/header.html [hunk 3]
```diff
@@ -59,87 +59,85 @@
...
-                            <a href="{{ url_for('portal_ui.other_portal_links') }}">Other Water Quality Portals</a>
+                            <a href="{{ url_for('portal_ui.other_portal_links') }}"> Other Water Quality Portals </a>
                         </li>
                         <li class="usa-nav__submenu-item">
-                            <a href="{{ url_for('portal_ui.publications') }}">Publications</a>
+                            <a href="{{ url_for('portal_ui.publications') }}"> Publications </a>
                         </li>
                         <li class="usa-nav__submenu-item">
-                            <a href="{{ url_for('portal_ui.contact_us') }}">Contact Us</a>
+                            <a href="{{ url_for('portal_ui.contact_us') }}"> Contact US</a>
```
Comment (abriggs-usgs) [inline server/wqp/portal_ui_blueprint/templates/partials/header.html:137]: Should this 'S' be lowercase?

**Review by abriggs-usgs — APPROVED** [review]
Thanks for cleaning that up. Just had a couple tiny comments.

File: server/wqp/portal_ui_blueprint/templates/partials/header.html (hunk 3 above)
Comment (mbucknell) [reply server/wqp/portal_ui_blueprint/templates/partials/header.html:137]: yes ... thanks

**Pull request merged**